tools/cpp/windows_cc_configure.bzl

```diff
@@ -308,8 +325,8 @@ def configure_windows_toolchain(repository_ctx):
                 "%{dbg_mode_debug}": "/DEBUG",
                 "%{fastbuild_mode_debug}": "/DEBUG",
                 "%{compilation_mode_content}": "",
-                "%{content}": _get_escaped_windows_msys_crosstool_content(repository_ctx),
-                "%{msys_x64_mingw_content}": _get_escaped_windows_msys_crosstool_content(repository_ctx, use_mingw = True),
+                "%{content}": _get_escaped_windows_msys_crosstool_content(repository_ctx, "mingw"),
+                "%{msys_x64_mingw_content}": _get_escaped_windows_msys_crosstool_content(repository_ctx, "msys"),
                 "%{opt_content}": "",
```

Comment:
Why switching these two content? Now `%{msys_x64_mingw_content}` will actually be replaced with msys toolchain content.

Reply:
Those were the remnants of my previous idea. I forgot to change them back. Done.